test-specifications/crypto-service/accountCreateTransaction.md

```diff
@@ -0,0 +1,439 @@
+  "result": {
+    "accountId": "0.0.12345",
+    "status": "SUCCESS"
+  }
+}
+```
+
+### **Memo:**
+
+- The memo associated with the account (UTF-8 encoding max 100 bytes).
+
+| Test no | Name                                                       | Input                                                                                                                         | Expected response                                                                                                                                                   | Implemented (Y/N) |
+|---------|------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates an account with a valid memo                       | key=<VALID_KEY>, memo="testmemo"                                                                                              | The account creation succeeds and the account's memo equals “testmemo”.                                                                                             | N                 |
+| 2       | Creates an account with an empty memo                      | key=<VALID_KEY>, memo=""                                                                                                      | The account creation succeeds and the account's memo is empty.                                                                                                      | N                 |
+| 3       | Creates an account with a memo that is 100 characters      | key=<VALID_KEY>, memo="This is a really long memo but it is still valid because it is 100 characters exactly on the money!!"  | The account creation succeeds and the account's memo equals "This is a really long memo but it is still valid because it is 100 characters exactly on the money!!". | N                 |
+| 4       | Creates an account with a memo that exceeds 100 characters | key=<VALID_KEY>, memo="This is a long memo that is not valid because it exceeds 100 characters and it should fail the test!!" | The account creation fails with a MEMO_TOO_LONG response code from the network.                                                                                     | N                 |
+
+#### JSON Request Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 99232,
+  "method": "createAccount",
+  "params": {
+    "key": "302e020100300506032b65700422042031f8eb3e77a04ebe599c51570976053009e619414f26bdd39676a5d3b2782a1d",
+    "memo": "testmemo"
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 99232,
+  "result": {
+    "accountId": "0.0.12345",
+    "status": "SUCCESS"
+  }
+}
+```
+
+### **Max Automatic Token Associations:**
```

Comment:
I propose to make it to follow 'memo' example.

Reply:
Could you elaborate a bit more?

Reply:
Max Automatic Token Associations tests: min, max, invalid.
Memo tests: valid, min, max, invalid.

So I'm just proposing make Max Automatic Token Associations tests more verbose by adding one test with a valid value 🙂

Reply:
Understood, added a test